cpp/example-control.cc

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+#include "zendar-api.h"
+#include "reqrep.pb.h"
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    printf("Usage: example-control URI");
+  }
+  zendar::ZendarReceiver rcv(argv[1]);
+  zendar::ZendarError error;
+  zen_proto::control::Response rep;
+  rcv.Connect();
+  rcv.Stop();
+  rcv.Status(rep);
+  if (rep.status().state() == zen_proto::control::RepStatus::READY) {
+    std::cout << "ZPU ready" << "\n";
+  } else {
+    std::cout << "ZPU not ready" << "\n";
+    return 0;
+  }
+  rcv.ListConfigurations(rep);
+  if (!rep.has_list_configurations()) {
+    std::cout << "Failed to list configurations";
+    return 0;
+  }
+  std::cout << "Available configurations:" << "\n";
+  for (auto config : rep.list_configurations().configurations()) {
+    std::cout << config << "\n";
+  }
+  return 1;
```

Comment:
doesn't a `0` return code indicate success?

Reply:
We can do it either way, this is a matter of convention.

Reply:
OK, I would suggest returning `0` in case customers copy / paste the code in their own apps.

Reply:
I can change it to 0, but the reasoning here doesn't make much sense to me. Whether 0 or 1 makes more sense then depends on the customers' convention, and I have seen both ways, even in our own code. In addition, they can't just copy-paste the whole program into their code, so I don't see any scenario where they would be copying the return code line.

Reply:
Anyway, I made the change.

Reply:
I think the convention is, '0' for success, anything other than `0` for an error. The way to interpret the error is application defined, but the '0' is always success.